chapters/open_research.md

```diff
@@ -11,58 +11,57 @@ Recommended skill level: low.
-        1. [Privacy](#Privacy)
-        2. [National and commercially sensitive data](#National_and_commercially_sensitive_data)
-4. [Open source software](#Open_source_software)
-    1. [What is open source software](#What_is_open_source_software)
-    2. [How running and contributing to open source software projects benefits you](#How_running_and_contributing_to_open_source_software_projects_benefits_you)
-        1. [Making your own work open source](#Making_your_own_work_open_source)
-        2. [Contributing to others' open source software projects](#Contributing_to_others_open_source_software_projects)
-    3. [How open source software benefits research](#How_open_source_software_benefits_research)
-    4. [How to run your own open source software project](#How_to_run_your_own_open_source_software_project)
-        1. [Readme](#Readme)
-        2. [Contributing guidelines](#Contributing_guidelines)
-        3. [Code of conduct](#Code_of_conduct)
-    5. [How to contribute to other's open source software projects](#How_to_contribute_to_others_open_source_software_projects)
-        1. [Anatomy of an open source software project](#Anatomy_of_an_open_source_software_project)
-        2. [Contribute your changes](#Contribute_your_changes)
-        3. [Looking for projects to contribute to and how to contribute to them](#Looking_for_projects_to_contribute_to_and_how_to_contribute_to_them)
-5. [Open hardware](#Open_hardware)
-    1. [Why open hardware?](#Why_open_hardware)
-    2. [Elements of an open source hardware project](#Elements_of_an_open_source_hardware_project)
-    3. [Open source hardware processes and practices](#Open_source_hardware_processes_and_practices)
-        1. [Designing your hardware](#Designing_your_hardware)
-        2. [Hosting your design file](#Hosting_your_design_files)
-        3. [Distributing open source hardware](#Distributing_open_source_hardware)
-6. [Open access](#Open_access)
-    1. [What is open access?](#What_is_open_access)
-        1. [Repositories and self-archiving](#Repositories_and_self_archiving)
-        2. [Open access publishing](#Open_access_publishing)
-    2. [Why does open access matter?](#Why_does_open_access_matter)
-    3. [Best practice for open access](#Best_practice_for_open_access)
-        1. [Self-archiving](#Self_archiving)
-        2. [Publication](#Publication)
-7. [Open notebooks](#Open_notebooks)
-8. [Open scholarship](#Open_scholarship)
-    1. [Open educational resources](#Open_educational_resources)
-    2. [Equity, Diversity, Inclusion](#Equity_Diversity_Inclusion)
-    3. [Citizen science](#Citizen_science)
-9. [Checklists](#Checklists)
-10. [What to learn next](#What_to_learn_next)
-11. [Further reading](#Further_reading)
-12. [Definitions/glossary](#Glossary)
-13. [Bibliography](#Bibliography)
-
-<a name="Summary"></a>
+1. [Summary](#summary)
+2. [How this will help you/ why this is useful](#why-this-is-useful)
```

Comment:
Hi @jmgate! Thank you for putting in this pull request.

I think this link will need to be:
```suggestion
2. [How this will help you/ why this is useful](#how-this-will-help-you-why-this-is-useful)
```